lib/web/apiserver.go

```diff
@@ -1397,6 +1397,7 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 			PreferredLocalMFA:  cap.GetPreferredLocalMFA(),
 			LocalConnectorName: localConnectorName,
 			PrivateKeyPolicy:   cap.GetPrivateKeyPolicy(),
+			MOTD:               cap.GetMessageOfTheDay(),
```

Comment:
there's a test for this that we should probably add to

Reply:
added in https://github.com/gravitational/teleport/pull/27813/commits/af6c5b6171a91e93670cf67a0b2ac365f12c97c7